src/app/workflow/versions/versions.component.html

```diff
@@ -134,6 +134,28 @@
       </td>
     </ng-container>
 
+    <ng-container matColumnDef="open" [matTooltip]="openTooltip">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        matTooltip="The version has a test parameter file with open data or requires no input files."
+      >
+        Open
+        <a
+          class="ds-green"
+          [href]="Dockstore.DOCUMENTATION_URL + '/faq.html#what-is-an-open-tool-or-workflow'"
```

Comment:
Does this exist?

Reply:
> Does this exist?

Not yet, but I have a ticket for it dockstore/dockstore#5402